tests/test_objects.py

```diff
@@ -2178,6 +2178,271 @@ def test_make_ingress_external_name():
     }
 
 
+@pytest.mark.parametrize(
+    'target,ip',
+    [
+        ('http://192.168.1.10:9000', '192.168.1.10'),
+        ('http://[2001:db8::dead:babe]:9000', '2001:db8::dead:babe'),
```

Comment:
Could you update this to be something nobody ends up reflecting on?

Reply:
And hopefully something slightly less morbid, at least for my sake? :D

Reply:
If you're talking about `dead:babe` thing, this is just a copy of existing test: https://github.com/jupyterhub/kubespawner/blob/acfc4d5efc364b8eff07efbe83da3904c3a80435/tests/test_objects.py#L1980

Reply:
:( can you update both?

Reply:
Ok, done